source/extensions/filters/http/jwt_authn/filter_config.cc

```diff
@@ -29,6 +37,34 @@ void FilterConfigImpl::init() {
           it.first, Verifier::create(it.second, proto_config_.providers(), *this));
     }
   }
+
+  for (const auto& it : proto_config_.requirement_map()) {
+    if (top_requirement_names_for_debug_.size() < TopRequirementNameForDebugSize) {
```

Comment:
Have unit test for this string size limit?

Reply:
Changed to print all names.  It will be unit-tested